src/EnergyPlus/HVACDXSystem.cc

```diff
@@ -1824,6 +1824,7 @@ namespace HVACDXSystem {
 
 									if ( ( TempSpeedOut - TempSpeedReqst ) < Acc ) {
 										SpeedNum = I;
+										FullOutput = TempSpeedOut; // save in case warning message is reported
```

Comment:
This and next are incidental changes for warning messages seen during testing. Just correcting them here since they are minor.